integration-tests/paypalpayflow_test.go

```diff
@@ -0,0 +1,145 @@
+package test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/BoltApp/sleet"
+	"github.com/BoltApp/sleet/common"
+	"github.com/BoltApp/sleet/gateways/paypalpayflow"
+	sleet_testing "github.com/BoltApp/sleet/testing"
+	"github.com/Pallinder/go-randomdata"
+)
+
+// TestPaypalAuth
+//
+// This should successfully create an authorization on Paypal Payflow
+func TestPaypalAuth(t *testing.T) {
```

Comment:
Great tests! Can we add one where authorization fails to make sure error handling works as expected?

Reply:
Ok